patches/third_party-blink-renderer-core-html-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/core/html/BUILD.gn b/third_party/blink/renderer/core/html/BUILD.gn
+index 27b124d4fb7dea76e30969585337f938964bd125..f1989af7a5e4dfc6fd04ddcc6fedeffeefd8e4be 100644
+--- a/third_party/blink/renderer/core/html/BUILD.gn
++++ b/third_party/blink/renderer/core/html/BUILD.gn
+@@ -463,6 +463,8 @@ blink_core_sources("html") {
+     "list_item_ordinal.h",
+     "media/autoplay_policy.cc",
+     "media/autoplay_policy.h",
++    "//third_party/blink/renderer/modules/permissions/permission_utils.cc",
```

Comment:
`blink/renderer/modules` depends on `blink/renderer/core` so if I add deps here will introduce circular deps

Reply:
we use functions of `third_party/blink/renderer/modules/permissions/permission_utils.h` in `third_party/blink/renderer/core/html/media/autoplay_policy.cc` for requesting permission.
So we need to add dependency for "//third_party/blink/renderer/modules/permissions:permissions"
Unfortunately, it will cause this circular dependency error
```
ERROR Dependency cycle:
  //third_party/blink/renderer/core:core ->
  //third_party/blink/renderer/core/html:html ->
  //third_party/blink/renderer/modules/permissions:permissions ->
  //third_party/blink/renderer/core:core
```

Reply:
but this will effectively build it twice and we don't want that. What build error do you get without it?

Reply:
```
Undefined symbols for architecture x86_64:
  "blink::ConnectToPermissionService(blink::ExecutionContext*, mojo::InterfaceRequest<blink::mojom::blink::PermissionService>)", referenced from:
      blink::AutoplayPolicy::IsAutoplayAllowedPerSettings() const in autoplay_policy.o
  "blink::CreatePermissionDescriptor(blink::mojom::PermissionName)", referenced from:
      blink::AutoplayPolicy::IsAutoplayAllowedPerSettings() const in autoplay_policy.o

```

Reply:
fixed in 2d275485a2f18c0af47c1b176fa8abff9cd9abe0